src/ophys_etl/modules/video/utils.py

```diff
@@ -431,26 +417,31 @@ def create_downsampled_video_h5(
     with h5py.File(output_path, 'w') as out_file:
         out_file.create_dataset('data',
                                 shape=(n_frames_out,
-                                       input_video_shape[1],
-                                       input_video_shape[2]),
+                                       frame_size[0],
+                                       frame_size[1]),
                                 chunks=(max(1, n_frames_out//100),
-                                        input_video_shape[1],
-                                        input_video_shape[2]),
+                                        frame_size[0],
+                                        frame_size[1]),
                                 dtype=float)
 
     mgr = multiprocessing.Manager()
     output_lock = mgr.Lock()
     validity_dict = mgr.dict()
     process_list = []
+
+    print(f'spatial filter {spatial_filter}')
```

Comment:
Maybe you meant to use the logger?

Reply:
...I think I actually meant to remove that print statement. I added that when I was implementing `spatial_filter` to make sure it got passed through. Thanks for catching that.